app/src/main/kotlin/com/simplemobiletools/notes/pro/activities/MainActivity.kt

```diff
@@ -241,7 +243,8 @@ class MainActivity : SimpleActivity() {
         }
     }
 
-    private fun isCurrentItemChecklist() = mAdapter?.isChecklistFragment(view_pager.currentItem) ?: false
+    private fun isCurrentItemChecklist() = mAdapter?.isChecklistFragment(view_pager.currentItem)
+            ?: false
```

Comment:
undo these formatting changes please, it is hard to see what really changed

Reply:
Please check now